nautobot_secrets_providers/templates/nautobot_secrets_providers/home.html

```diff
@@ -33,6 +33,7 @@ <h1>{% block title %}Secrets Providers Home{% endblock %}</h1>
                     </tr>
                     <tr>
                         <td><a href="https://www.vaultproject.io" rel="nofollow">HashiCorp Vault</a></td>
+                        <td><a href="https://www.vaultproject.io/docs/secrets/kv/kv-v1" rel="nofollow">K/V Version 1</a></td>
                         <td><a href="https://www.vaultproject.io/docs/secrets/kv/kv-v2" rel="nofollow">K/V Version 2</a></td>
```

Comment:
This looks wrong to me, should it instead be?

```suggestion
                        <td><a href="https://www.vaultproject.io/docs/secrets/kv/kv-v1" rel="nofollow">K/V Version 1</a>
                            <br><a href="https://www.vaultproject.io/docs/secrets/kv/kv-v2" rel="nofollow">K/V Version 2</a></td>
```